pyfr/integrators/writers.py

```diff
@@ -1,17 +1,16 @@
 # -*- coding: utf-8 -*-
 
-from abc import abstractmethod
 import itertools as it
 import os
-
+import h5py
+from mpi4py import MPI
```

Comment:
Do not globally import mpi4py.  (It is one of the few exceptions to the rule about avoiding local imports.)